site/faq/README.md

```diff
@@ -51,6 +51,47 @@ resource configuration provides a number of desirable properties:
 7. it **supports display in UI and tools** which use either OpenAPI or the
    YAML/JSON directly.
 
+
+### Does kpt and kustomize work together?
+
+Yes!  Because both tools subscribe to template/DSL free approach to configuraiton it's possible to use both tools together.  We have created a 
+[kustomize solution] which shows how you can leverage both of these tools
+in your environment.
+
+#### Kustomize background
+[Kustomize] is a CNCF project that is a part of Kubernetes.  It's included in 
+the `kubectl` in order to allow users to customize their configurations without introducing templates.   Kustomize was started by Googlers and still has 
+large contributions by Google engineers.  A lot of the Google engineers working
+on kpt also work on kustomize and a lot of the underlying libraries are shared.
+
+While kpt and kustomize support customization of configuration yaml there are 
```

Comment:
Say "KRM manifests" instead of "configuration yaml"

Reply:
done